nano/secure/ledger.cpp

```diff
@@ -574,8 +576,6 @@ void ledger_processor::receive_block (nano::receive_block & block_a)
 						result = ledger.block_or_pruned_exists (transaction, block_a.hashables.source) ? nano::block_status::progress : nano::block_status::gap_source; // Have we seen the source block already? (Harmless)
 						if (result == nano::block_status::progress)
 						{
-							auto info = ledger.account_info (transaction, account);
-							debug_assert (info);
 							result = info->head == block_a.hashables.previous ? nano::block_status::progress : nano::block_status::gap_previous; // Block doesn't immediately follow latest block (Harmless)
```

Comment:
Why do we need this check and why is it returning gap_previous?

Reply:
The setting f variable info moved upwards but this check, which is now redundant, was left behind.